amanuensis/resources/request/__init__.py

```diff
@@ -40,6 +37,21 @@ def get(logged_user_id, consortium=None):
 
 
 def get_by_id(logged_user_id, request_id):
+    
+    if environ.get('GEN3_DEBUG', '').lower() == 'true' and int(request_id) == 12345:
```

Comment:
I think we can remove the request_id == 12345 in the condition for the DEBUG here, the DEBUG flag should be enough. Am I missing anything?

Reply:
In dev testing, GEN3_DEBUG=True is the norm, but if you want to test legit request data, this block would get in the way.  Limiting to request_id=12345 avoids potential problems later.  I could also just comment the whole block, but leave it in place.